src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,65 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Class implementing stack
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int NUMBERS_QUANTITY = 12;
+	private static final int EMPTY_STACK_INDICATOR = -1;
+	private static final int FULL_STACK_INDICATOR = 11;
 
-	public int total = -1;
+	private int[] numbers = new int[NUMBERS_QUANTITY];
 
+    private int total = EMPTY_STACK_INDICATOR;
+
+    /**
+     * @return amount of values currently on stack
+     */
+    public int getTotal() {
+        return total;
+    }
+
+    /**
+     * Put value on stack
+     * @param in value to put
+     */
 	public void countIn(int in) {
 		if (!isFull())
 			numbers[++total] = in;
 	}
 
-	public boolean callCheck() {
-		return total == -1;
+    /**
+     * Checks if stack is empty
+     * @return true if stack is empty
+     */
+	boolean isEmpty() {
```

Comment:
Changing an interface of DefaultCountingOutRhymer is forbidden (callCheck should remain). It is the API of this library!